soc/openhwgroup/cva6/cv64a6/Kconfig.soc

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_CV64A6
+	bool
+	select SOC_FAMILY_OPENHWGROUP_CVA6
+	help
+	  CVA6 64-bit RISC-V CPUs, commonly used as soft cores on FPGAs.
+	  This configuration works with the upstream project (https://github.com/openhwgroup/cva6)
+	  as well as the forked project with support for the Xilinx AXI Ethernet Subsystem
+	  (https://github.com/cispa/CVA6-Vivado-Project-with-Xilinx-AXI-Ethernet).
+
+config SOC_CV64A6_IMAFDC
+	bool "CV64A6 with imafdc instruction sets"
```

Comment:
no selections here

Reply:
Done (moved to Kconfig, I hope that is the correct place)

Reply:
Actually, all >200 `select SOC_FAMILY_*` and `select SOC_SERIES_*` statements in the codebase appear to be in `Kconfig.soc` files.